mssw/My-List-Util/lib/Test/My/List/Util.pm

```diff
@@ -0,0 +1,227 @@
+package Test::My::List::Util;
+
+use 5.006;
+use strict;
+use warnings;
+
+use Exporter qw(import);
+use vars qw(@EXPORT $VERSION);
```

Comment:
You `use vars` `$VERSION` but then declare it with `our $VERSION` below - you don't need both `use vars` and `our`.